### PR TITLE
feat: Describe session tokens

### PR DESCRIPTION
**Pull request by blastrock**

### CODE REVIEW

File: features.md [hunk 1]
```diff
@@ -62,6 +62,20 @@ This allows the *device* to decrypt any encrypted *data* shared with the *user*
...
+### Functional perimeter
+
+When a *user*  performs *identity verification* they can also request a *session token*.
+A *session token* can be used by the *application server* to verify that an *identity verification* took place, and to check what *verification method* was used.
+
+This allows the *application server* to implement policies like multi-factor authentication.
+
+### Access perimeter
+
+This operation requires that the device generate a signed *session certificate* when performing *identity verification*.
```
Comment (blastrock) [inline features.md:76]: generates

File: features.md [hunk 2]
```diff
@@ -62,6 +62,20 @@ This allows the *device* to decrypt any encrypted *data* shared with the *user*
...
+
+When a *user*  performs *identity verification* they can also request a *session token*.
+A *session token* can be used by the *application server* to verify that an *identity verification* took place, and to check what *verification method* was used.
+
+This allows the *application server* to implement policies like multi-factor authentication.
+
+### Access perimeter
+
+This operation requires that the device generate a signed *session certificate* when performing *identity verification*. 
+This signed *session certificate* is stored on the *Tanker server* and corresponds to the *session token*.
```
Comment (blastrock) [inline features.md:77]: and its hash is the session token*

File: features.md [hunk 3]
```diff
@@ -62,6 +62,20 @@ This allows the *device* to decrypt any encrypted *data* shared with the *user*
 
 This operation can for example be done with a *verification ke*y, which is usually stored on the *Tanker server*, protected by the *verification method* set up at the identity registration.
 
+## Session tokens
+
+### Functional perimeter
+
+When a *user*  performs *identity verification* they can also request a *session token*.
```
Comment (AlexandreBossard) [inline features.md]: an identity verification ?

File: features.md [hunk 4]
```diff
@@ -62,6 +62,20 @@ This allows the *device* to decrypt any encrypted *data* shared with the *user*
...
+A *session token* can be used by the *application server* to verify that an *identity verification* took place, and to check what *verification method* was used.
```
Comment (AlexandreBossard) [inline features.md]: and to -> and also

File: protocol.md [hunk 5]
```diff
@@ -96,6 +96,23 @@ Steps to register a new *device*:
...
 
+### Session Tokens
+
+Prerequisite: The user is registered on the application.
+
+A *session token* can be obtained when a user creates their first device, or at any time later when proving ownership of a  [Verification Method](#verification-methods).
+
+Steps to obtain and check a *session token*:
+1. The *user* uses one the [Verification Method]s available to them, and requests a *session token* through the API
+1. *Tanker Core* generates a random nonce and sends it to the *Tanker server* along with the verification
```
Comment (AlexandreBossard) [inline protocol.md]: do nounce are not random by definition ?

Comment (tux3) [reply protocol.md]: Not necessarily as long as they're only used once, but I can remove the word "random"

**Review by AlexandreBossard — APPROVED** [review]

**Pull request merged**